Sources/FoundationEssentials/Calendar/Calendar_Gregorian.swift

```diff
@@ -192,10 +193,20 @@ internal final class _CalendarGregorian: _CalendarProtocol, @unchecked Sendable
             self.gregorianStartDate = Date(timeIntervalSince1970: -12219292800) // 1582-10-15T00:00:00Z
         }
 
-        self.locale = locale
+        // We do not store the Locale here, as Locale stores a Calendar. We only keep the values we need that affect Calendar's operation.
+        if let locale {
+            localeIdentifier = locale.identifier
+            localePrefs = locale.prefs
+        } else {
+            localeIdentifier = ""
+            localePrefs = nil
+        }
+
         self.localeIdentifier = locale?.identifier ?? ""
```

Comment:
Is this line effectively duplicate of the lines above that set `localeIdentifier`?

Reply:
oh yes. This should be removed